docs/source/quick_start.rst

```diff
@@ -32,26 +35,37 @@ A simple example of this functionality is::
    plt.plot(res['time'], res['inertia1.phi'])
    plt.show()
 
-An example of setting up and executing a series of simulations on a server and returning the plot trajectories::
+A simple example using the :ref:`Class name based workflow <2. Class name based workflow>` is shown below::
 
-   from modelon.impact.client import Client, Range
+   from modelon.impact.client import Client
 
    client = Client(url=<impact-domain>)
    workspace = client.create_workspace(<workspace-name>)
 
    # Choose analysis type
    dynamic = workspace.get_custom_function('dynamic')
 
-   # Compile model
+   # Get model class
    model = workspace.get_model("Modelica.Blocks.Examples.PID_Controller")
-   fmu = model.compile(
-      compiler_options=dynamic.get_compiler_options().with_values(
-         generate_html_diagnostics=True
-      )
-   ).wait()
 
    # Execute experiment
-   experiment_definition = fmu.new_experiment_definition(
+   experiment_definition = model.new_experiment_definition(dynamic)
+   exp = workspace.execute(experiment_definition).wait()
+
+   # Plot Trajectory
+   import matplotlib.pyplot as plt
+
+   case = exp.get_case('case_1')
+   res = case.get_trajectories()
+   plt.plot(res['time'], res['inertia1.phi'])
+   plt.show()
+
+An example of setting up and executing a series of simulations on a server and returning the plot trajectories::
+
+   from modelon.impact.client import Range
+

```

Comment:
I think these quick start code examples should be complete to help new users, so you would need to add the lines:
```
client = Client(url=<impact-domain>)
workspace = client.create_workspace(<workspace-name>)
dynamic = workspace.get_custom_function('dynamic')
```